src/api/status/src/server.js

```diff
@@ -114,8 +136,20 @@ service.router.get(process.env.PATH_PREFIX || '/', async (req, res) => {
 });
 
 // Get Build page
-service.router.get(`${process.env.PATH_PREFIX || ''}/build`, (req, res) => {
-  res.render('builds', { active_build_log: true, headers: { title: 'Telescope Build log' } });
+service.router.get(`${process.env.PATH_PREFIX || ''}/build`, async (req, res) => {
+  let distJsPath = '';
+
+  try {
+    distJsPath = await getDistJsPath('public/js/pages/build.js');
+  } catch (error) {
+    console.error(error);
```

Comment:
This is a bit weak.  First, use `logger` from Satellite and log it properly.  Second, if we don't have this value, should we have crashed earlier with an error (e.g., on startup)?

Reply:
It won't crash, just 404 the script file. How about
```js
try {
 const distJsPath = await getDistJsPath();
 return res.render();
} catch(error) {
  log(error);
}

return res.send("Fail to render /build");
```

Reply:
`logger.warn({ error }, 'unable to get dist/js path')`

Reply:
For the crashing if it fails. Can we just send back a response instead of rendering the page? (like in the snipppet)

Reply:
Sure, just so it doesn't take the server down.